tower.yml

```diff
@@ -0,0 +1,5 @@
+reports:
+  multiqc_report.html:
+    display: "MultiQC HTML report"
+  samplesheet.csv:
+    display: "Auto-created samplesheet with collated metadata and FASTQ paths"
```

Comment:
It'll be nice to have all small files created by the pipeline included there

Reply:
@jfy133  Maybe next release?

Reply:
I have no idea what this file does :grimacing: so yes (although I think I do...), next release!

Reply:
Agreed. I have created an issue #373 to address it in the next release.